src/routes/components/popover/+page.md

```diff
@@ -0,0 +1,141 @@
+<script lang="ts">
+    import Popover from "$lib/components/Popover.svelte";
+    import {IconMenu, IconAccount, IconExit, IconWallet, IconLightMode, IconDarkMode} from "$lib/icons";
+    let visible = false;
+    let button: HTMLButtonElement | undefined;
+    let visible2 = false;
+    let button2: HTMLButtonElement | undefined;
+    let visible3 = false;
+    let button3: HTMLButtonElement | undefined;
+    let selected1: boolean = true;
+</script>
+
+# Popover
+
+A popover is a floating container that is rendered over the content
+next to an anchor — commonly a button — which initiates its display.
+<br />
+
+# Showcase
+
+<div id="display">
+    <button 
+        class="primary"
+        bind:this={button2} 
+        on:click={() => (visible2 = !visible2)}
+    >
+        Open Popover
+    </button>
+    <Popover bind:visible={visible2} anchor={button2}>
+        This is a popover!
+    </Popover>
+    <button
+        data-tid="account-menu"
+        class="icon-only toggle"
+        bind:this={button}
+        on:click={() => (visible = !visible)}
+    >  
+        <IconMenu size={40}/>
+    </button>
+    <Popover bind:visible anchor={button} closeButton>
+        <div class="account">
+            <span class="account_icon">
+                <IconAccount size={34} />
+            </span>Account
+        </div>
+        <div class="account">
+            <span class="account_icon">
+                <IconWallet size={34}/>
+            </span>Wallet
+        </div>
+        <div class="account">
+            <span class="account_icon">
+                <IconExit size={34}/>
+            </span>Logout
+        </div>
+    </Popover>
+    <button
```

Comment:
you can simplify and remove the showcase "Apparence".

Once this PR added, I'll take care of adding the global dark/light mode theme switcher and capatibility to the gix-components.